spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -976,6 +974,115 @@ public Message<O> process(Message<I> msg) {
 The implementation is reponsible for enforcing the acknowledgement strategy defined by the user when the `@Acknowledgement` policy is used. 
 If the annotation is not used, the default policy must be enforced.
 
+=== Connector
+
+Reactive Messaging connects matching `@Incoming` and `@Outgoing` running inside the same application.
+Besides, it maps specific _channels_ to _external_ technologies such as Apache Kafka, MQTT, Web Sockets, or JMS. 
+It means that Reactive Messaging can receive messages from virtually any messaging technology and dispatch messages to any messaging technology.
+The component responsible for this role is called _connector_. 
+
+==== Connector concepts
+
+Each _connector_ is responsible for a specific technology. 
+A connector can:
+
+* act as a _Publisher_, meaning it retrieves or receives messages from an external messaging technology. The messages will then be sent to a method annotated with `@Incoming`
+* act as a _Subscriber_, meaning it dispatches messages to an external messaging technology. The messages are received from a method annotated with `@Outgoing`
+* handle both directions
+
+It's essential that connectors implement the back-pressure protocol defined by the Reactive Streams specification.
+
+A _connector_ is implemented as a CDI Bean, generally _application_ scoped implementing:
```

Comment:
Does it have to implement 1 of or both of these interfaces?

Reply:
Depends on your connector. Generally, both except... when you integrate read-only mechanism (dbus) or write-only systems (machine learning).

Adding a sentence about this.